src/Microsoft.AspNetCore.Mvc.ViewFeatures/project.json

```diff
@@ -1,5 +1,9 @@
 {
-  "description": "View Rendering features for the MVC runtime",
+  "description": "ASP.NET Core MVC view rendering features. Contains common types used in most MVC applications as well as view rendering features such as view engines, views, view components, and HTML helpers.\r\nCommonly used types:Microsoft.AspNetCore.Mvc.Controller\r\nMicrosoft.AspNetCore.Mvc.ValidateAntiForgeryTokenAttribute\r\nMicrosoft.AspNetCore.Mvc.ViewComponent",
```

Comment:
Not sure `ViewComponent` fits since commonly-reported bugs aren't necessarily the same as commonly-used types :smiling_imp: Might be worth mentioning `IHtmlHelper`.